srv/modules/runners/sharedsecret.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+
+
+def show():
+    filename = '/etc/salt/master.d/sharedsecret.conf'
+    if not os.path.exists(filename):
+        return None
+    with open(filename, 'r') as fhandler:
+        line = fhandler.readline()
+        idx = line.find(':')
```

Comment:
Maybe you should add a comment why it is necessary to parse and split the line. I don't know how the file  looks like, so an example would be helpful to understand the code.